rabbitmq/README.md

```diff
@@ -118,7 +95,29 @@ Configuration Options:
                 pattern: "="
     ```
 
-4. [Restart the Agent][10].
+4. [Restart the Agent][8].
+
+#### Containerized
+
+For containerized environments, see the [Autodiscovery Integration Templates][9] for guidance on applying the parameters below.
+
+##### Metric collection
+
+| Parameter            | Value                            |
+| -------------------- | -------------------------------- |
+| `<INTEGRATION_NAME>` | `rabbitmq`                       |
+| `<INIT_CONFIG>`      | blank or `{}`                    |
+| `<INSTANCE_CONFIG>`  | `{"host":"%%host%%:15672/api/"}` |
```

Comment:
Shouldn't this be `rabbitmq_api_url` instead of `host`?

Reply:
indeed it's a miss

Reply:
```suggestion
| `<INSTANCE_CONFIG>`  | `{"rabbitmq_api_url":"%%host%%:15672/api/"}` |
```